### PR TITLE
Update test-readme.md

### PR DESCRIPTION
**Pull request by elagrog**
I've changed the readme text to be my cv.

### CODE REVIEW

**Comment by elagrog** [comment]
@lsnovich, I cannot seem to add your user name as a reviewer. I hope you'll see this.

**Review by lsnovich — APPROVED** [review]
Nice work!
:smiley_cat: 
you can merge this PR

File: test-readme.md [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
-* Trial and error
-* Copying examples and seeing their output.
-I will use [this reference](https://guides.github.com/features/mastering-markdown/) as a guide.
-
-One day **soon** I will be a markdown master.
+# Ela Greenberg
+
+_Jerusalem, Israel_
+
+[LinkedIn] (www.linkedin.com/in/elagreenberg)
```
Comment (lsnovich) [inline test-readme.md:5]:
```suggestion
[LinkedIn](www.linkedin.com/in/elagreenberg)
```

File: test-readme.md [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
+
+
+English language editor and aspiring technical communicator
```
Comment (lsnovich) [inline test-readme.md:8]:
```suggestion
English language editor and aspiring technical communicator.
```

File: test-readme.md [hunk 3]
```diff
@@ -1,13 +1,42 @@
...
+## Summary 
+* Over 10 years experience of editing and proofreading texts
+* Specializes in Middle East and Islamic studies, international relations policy fields, and non-profits
+* Works mostly with ESL authors
+* Native English speaker, fluent Hebrew, and basic Arabic
+
+## Experience
+**Dr. Ela Greenberg Editorial Services**
+_Copyeditor_
+* Engages in copyediting and proofreading
```
Comment (lsnovich) [inline test-readme.md:22]: change tense to imperative

File: test-readme.md [hunk 4]
```diff
@@ -1,13 +1,42 @@
...
+* Copyedits, proofreads, and prepares for publication 1600 pages per year for the Institute for National Security Studies (INSS)—including
```
Comment (lsnovich) [inline test-readme.md:23]: change your emdash to a unicode

Comment (lsnovich) [reply test-readme.md:23]: ``&mdash;``

File: test-readme.md [hunk 5]
```diff
@@ -1,13 +1,42 @@
...
+
+
+## Summary 
+* Over 10 years experience of editing and proofreading texts
+* Specializes in Middle East and Islamic studies, international relations policy fields, and non-profits
+* Works mostly with ESL authors
+* Native English speaker, fluent Hebrew, and basic Arabic
+
+## Experience
+**Dr. Ela Greenberg Editorial Services**
```
Comment (lsnovich) [inline test-readme.md:20]: please add dates

**Review by lsnovich — CHANGES_REQUESTED** [review]
make some small changes please

**Comment by lsnovich** [comment]
What I suggest you do is
1. make the changes I asked for
2. Open a new PR

File: test-readme.md (hunk 2 above)
Comment (elagrog) [reply test-readme.md:8]: Corrected.

File: test-readme.md (hunk 3 above)
Comment (elagrog) [reply test-readme.md:22]: why? It doesn't make sense. I'm not telling somebody to copyedit.

File: test-readme.md (hunk 5 above)
Comment (elagrog) [reply test-readme.md:20]: corrected

File: test-readme.md (hunk 3 above)
Comment (elagrog) [reply test-readme.md:22]: I changed it anyway, although I would not write a cv that way.

File: test-readme.md (hunk 4 above)
Comment (elagrog) [reply test-readme.md:23]: changed.

**Pull request merged**